src/style.css

```
@@ -1,3 +1,73 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --main-color: #00acdc;
+}
+
+header {
```

Use the classes selector
```suggestion
.header {
```